source/objects/responseClass.m

```diff
@@ -318,6 +318,141 @@ function plotForces(obj,bodyNum,comp)
             clear t FT FE FRD FR FV FM i
         end
         
+        function plotWaves(obj,it,tsf,g,axis_lims,body,waves,save)
+            % This method plots the wave elevation and body geometry over
+            % time to visualize the waves and response
+            %
+            % Parameters
+            % ------------
+            %     it : float
```

Comment:
Replace with ``simu.dtOut`` throughout the function. Note that the wave elevation is a pre-processing step so simu.dtOut is not used, it is instead sampled at simu.dt. However, the body's position response will have a sample time of simu.dtOut.  If simu.dtOut~=simu.dt, the function will break.